.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: purescript-contrib/setup-purescript@main
         with:
           purescript: "unstable"
+        purs-tidy: "stable"
```

Comment:
If we're automatically adding `purs-tidy` as part of these releases (which I think we should do!) then we should also commit a standard `.tidyrc.json` file. I'd recommend something like this:

```json
{
  "importSort": "ide",
  "importWrap": "source",
  "indent": 2,
  "operatorsFile": null,
  "ribbon": 1,
  "typeArrowPlacement": "first",
  "unicode": "never",
  "width": null
}
```

Reply:
If you _did_ do this, then it didn't get committed because of the `.gitignore` file:

https://github.com/purescript/purescript-prelude/blob/5f1ba9faea5d9ab6fba2c74cc1ba0aaeda8b465b/.gitignore#L1

You'll want to append `!/.tidyrc.json` and a newline to the end of the gitignore.

Reply:
Ah, thanks. Fixed that in my script.

Reply:
My previous comment was for the `stable` -> `latest` fix, not the rest of this thread. Good point. I had forgotten about that.